webapp/src/Entity/Contest.php

```diff
@@ -381,6 +381,11 @@ public function getShortname(): string
         return $this->shortname;
     }
 
+    public function getShortDesc(): string
```

Comment:
I’d be more of a fan to use getShortDescription

Reply:
I agree, will fix it.